### DISCUSSION TITLE
Possible to apply DateRangeFilter on relationship?

### DISCUSSION THREAD

**Discussion by Jszul** [rappasoft/laravel-livewire-tables#1745] [Q&A]
Attempting to filter results by date where the date lives in a parent model, but I'm getting this error in the browser console:
 livewire.js?id=44144c23:3884 Livewire Entangle Error: Livewire property ['filterComponents.exam.test.date'] cannot be found on component: ['tests-table']

`<?php

namespace App\Livewire;

use App\Models\TestGrade;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Support\Facades\Log;
use Rappasoft\LaravelLivewireTables\DataTableComponent;
use Rappasoft\LaravelLivewireTables\Views\Column;
use Rappasoft\LaravelLivewireTables\Views\Filters\DateRangeFilter;

class TestsTable extends DataTableComponent
{
    protected $model = TestGrade::class;

    public function configure(): void
    {
        $this->setPrimaryKey('id')
            ->setFilterLayoutSlideDown();

        $this->addAdditionalSelects(['employee.first_name as forename']);
        $this->addAdditionalSelects(['employee.last_name as surname']);
    }

    public function filters(): array
    {
        return [
           DateRangeFilter::make('Test Date', 'exam.test.date')
                ->filter(function (Builder $builder, array $dateRange) {
                    if (isset($dateRange['minDate']) && isset($dateRange['maxDate'])) {
                        $builder->whereBetween('exam.test.date', [$dateRange['minDate'], $dateRange['maxDate']]);
                    }
                }),
        ];
    }

    public function columns(): array
    {
        return [
            Column::make("Employee Name", 'employee.first_name')
                ->format(
                    fn ($value, $row, Column $column) => $row->forename . ' ' . $row->surname
                ),
            Column::make("result", "result")
                ->sortable(),
            Column::make('test code', 'exam.test_code')
                ->sortable(),
            Column::make('District', 'exam.test.district')
                ->sortable(),
            Column::make('admininstrator title', 'exam.test.primaryAdministrator.title')
                ->sortable(),
            Column::make('Date', 'exam.test.date')
                ->sortable(),

        ];
    }
}
`

**Comment by lrljoe** [comment]
Don't give the filter a dot notated name (second parameter)

Also, the filter callback references the table name, and field, not the Eloquent relationship.
